### PR TITLE
feat!: prepended OneTrust to tracking integrations loads

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Adds OneTrust initialization to the `initializeTrackingIntegrations` call per the [Delay Loading OneTrust RFC](https://www.notion.so/Delay-Loading-OneTrust-acaceab477744c458b76d96ec9cb6bff).

I'm working on testing it out directly in the monolith with a feature flag now: https://github.com/codecademy-engineering/Codecademy/pull/24430

This is also being tested in portal-app: https://github.com/codecademy-engineering/portal-app/pull/39

### Notes

Removes any runtime dependency on React from the tracking package. Yay, portability!

### PR Checklist

- ~[ ] Related to designs:~
- [x] Related to JIRA ticket: WEB-1422
- [x] I have run this code to verify it works
- ~[ ] This PR includes unit tests for the code change~

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@1.0.0-alpha.1a96c1.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@1.0.0-alpha.f35bab.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://609c3bcc063003272ec8b52e--gamut-preview.netlify.app>https://609c3bcc063003272ec8b52e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/609c3bcc063003272ec8b52e>Deploy Logs</a></p>

File: packages/tracking/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # `@codecademy/tracking`
 
-This package contains our user "telemetry" (tracking metcics such as `click` and `visit`) integrations that are shared across multiple web properties.
+This package contains our user "telemetry" (tracking metrics such as `click` and `visit`) integrations that are shared across multiple web properties.
```
Comment (borisonr) [inline packages/tracking/README.md:3]: 😄

**Review by borisonr — APPROVED** [review]
seems reasonable to me as long as we're cool with legal/compliance and are super vigilant to not add any cookies prior to loading this

File: packages/tracking/README.md (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/tracking/README.md:3]: thakns

File: packages/tracking/src/integrations/onetrust.ts [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    };
+  });
+};
+
+const rawStyles = `
+:root {
+  --brand-purple: #6400e4;
+  --color-gray-500: #828285;
+  --color-white: #fff;
+}
```
Comment (codecaaron) [inline packages/tracking/src/integrations/onetrust.ts:45]: Up for removing these in favor of `theme.colors` variables before we add this as a shared package?

**Review by codecaaron — APPROVED** [review]
LGTM but small request on the colors.

**Review by dreamwasp — APPROVED** [review]
+1 to Aaron's request, everything looks good to me + the documentation/comments you added make everything here 1000% clearer

File: packages/tracking/src/integrations/onetrust.ts (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/tracking/src/integrations/onetrust.ts]: Talking on Slack: https://codecademy.slack.com/archives/CQ98K8PT6/p1621269048136300

For now, maybe just namespacing them?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@1.0.0-alpha.99d5ec.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a29d4c30595c009d0b46f8--gamut-preview.netlify.app>https://60a29d4c30595c009d0b46f8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a29d4c30595c009d0b46f8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/tracking@1.0.0-alpha.f35aad.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60abb17deedff70dcfc738e8--gamut-preview.netlify.app>https://60abb17deedff70dcfc738e8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60abb17deedff70dcfc738e8>Deploy Logs</a></p>

**Pull request merged**